diff --git a/lib/borders.stylex.js b/lib/borders.stylex.js
index ec22e248a380d69237e2b61f4c9fd21a05a68093..6ab7523793f3581091425f45f9bcc4e54a7d1356 100644
--- a/lib/borders.stylex.js
+++ b/lib/borders.stylex.js
@@ -33,9 +33,9 @@ export const borders = defineVars({
   // radiusBlob4: '63% 37% 37% 63% / 43% 37% 63% 57%',
   // radiusBlob5: '49% 51% 48% 52% / 57% 44% 56% 43%',
   radiusConditional1: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius1})`,
-  radiusConditional2: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius2}))`,
-  radiusConditional3: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius3}))`,
-  radiusConditional4: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius4}))`,
-  radiusConditional5: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius5}))`,
-  radiusConditional6: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius6}))`
+  radiusConditional2: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius2})`,
+  radiusConditional3: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius3})`,
+  radiusConditional4: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius4})`,
+  radiusConditional5: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius5})`,
+  radiusConditional6: `clamp(0px, calc(100vw - 100%) * 1e5, ${radius6})`
 });
